packages/configurationmap/stories/HashMap.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { withReadme } from "storybook-readme";
+import { HashMap } from "../index";
+import { configurationMapStoryWrapper } from "./helpers/ConfigurationMapStoryWrapper";
+
+const readme = require("../README.md");
+
+storiesOf("ConfigurationMap/HashMap", module)
+  .addDecorator(withReadme([readme]))
+  .addDecorator(configurationMapStoryWrapper)
+  .add("default", () => (
+    <HashMap
+      hash={{
+        name: "Jane Doe",
+        role: "UX Designer",
+        city: "San Francisco",
+        state: "CA",
+        zipcode: 95125
+      }}
+    />
+  ))
+  .add("with nested objects", () => (
+    <HashMap
+      hash={{
+        name: "Jane Doe",
+        role: "UX Designer",
+        city: "San Francisco",
+        state: "CA",
+        zipcode: 95125
+      }}
+    />
+  ))
+  .add("with headline", () => (
+    <HashMap
+      hash={{
+        name: "Jane Doe",
+        role: "UX Designer",
+        city: "San Francisco",
+        state: "CA",
+        zipcode: 95125
+      }}
+      headline="Jane Doe Info"
+    />
+  ))
+  .add("with nested objects", () => (
+    <HashMap
+      hash={{
+        name: "Jane Doe",
+        role: "UX Designer",
+        city: "San Francisco",
+        state: "CA",
+        zipcode: 95125,
+        Favorites: {
```

Comment:
Is there a way to specify `renderKeys` for nested keys?

Reply:
No, it only goes one level deep. Do you think this can be a follow-up ticket? Or changed in this PR?

Reply:
Follow up would be fine - it's probably not a terribly common use case!